lib/fform/renderfform.flow

```diff
@@ -189,7 +189,9 @@ renderFForm(rform : FForm, zorder : Transform<[int]>) -> FRenderResult {
 				renderFParagraph(text, style);
 			}
 		}
-		FPicture(url_, wh, style): {
+		FPicture(url2, wh, style2): {
+			f = FPictureApplyJwtExtraction(FPicture(url2, wh, style2));
```

Comment:
Also, let's make it FPictureApplyJwtExtraction(rForm) and don't create additional object.